tests/Utils/Models/Post.php

```diff
@@ -30,6 +30,11 @@ public function user(): BelongsTo
         return $this->belongsTo(User::class);
     }
 
+    public function activity(): MorphMany
```

Comment:
This needs a `@property-read`.

Reply:
What i meant is that the PHPDoc of the model is missing an annotation that makes sure that the magic relation access is typesafe.